webapp/src/Controller/SecurityController.php

```diff
@@ -145,10 +145,11 @@ public function registerAction(
             if ($this->config->get('show_affiliations')) {
                 switch ($registration_form->get('affiliation')->getData()) {
                     case 'new':
+                        $shortNameLength = 32;
```

Comment:
This variable is introduced but unused.

Reply:
FYI, the shortname is already truncated with a hard `substr(,0,32)` in the Entity/TeamAffiliation class. So the form validation plus that should ensure it's never too long.